common/scala/src/whisk/common/Logging.scala

```diff
@@ -149,10 +150,14 @@ object LoggingMarkers {
     val INVOKER_ACTIVATION_RUN_START = LogMarkerToken(invoker, activationRun, start)
     val INVOKER_ACTIVATION_RUN_DONE = LogMarkerToken(invoker, activationRun, finish)
 
+    val INVOKER_ACTIVATION_START = LogMarkerToken(invoker, activation, start)
     val INVOKER_ACTIVATION_END = LogMarkerToken(invoker, activation, finish)
-    val INVOKER_FAILED_ACTIVATION = LogMarkerToken(invoker, activation, error)
+    val INVOKER_ACTIVATION_ERROR = LogMarkerToken(invoker, activation, error)
 
     private val recordActivation = "recordActivation"
     val INVOKER_RECORD_ACTIVATION_START = LogMarkerToken(invoker, recordActivation, start)
     val INVOKER_RECORD_ACTIVATION_DONE = LogMarkerToken(invoker, recordActivation, finish)
+
+    val CLOUDANT_CACHE_HIT = LogMarkerToken(cloudant, "cacheHit", start)
+    val CLOUDANT_CACHE_MISS = LogMarkerToken(cloudant, "cacheMiss", start)
```

Comment:
We should call these `DATABASE_CACHE_HIT` and thelike since that interface is not Cloudant specific